build.zig

```diff
@@ -513,16 +526,34 @@ pub inline fn addGLFWSources(b: *std.Build, c_lib: *std.Build.Step.Compile, targ
 	}
 }
 
+pub fn addMiniaudioAndStbVorbis(b: *std.Build, c_lib: *std.Build.Step.Compile, flags: []const []const u8, replace_tool: *std.Build.Step.Compile) void {
+	const miniaudio = b.dependency("miniaudio", .{});
+	c_lib.addIncludePath(miniaudio.path(""));
+
+	c_lib.installHeader(miniaudio.path("extras/stb_vorbis.c"), "stb/stb_vorbis.h");
+	const miniaudioHeaderInstall = b.addInstallFile(miniaudio.path("extras/miniaudio_split/miniaudio.h"), "include/miniaudio.h");
+
+	// Patch miniaudio.h to avoid "loop dependency" issues (see: https://github.com/ziglang/zig/issues/12325)
+	const miniaudioHeaderPath = b.pathJoin(&.{"zig-out", "include", "miniaudio.h"});
```

Comment:
Please just write the path. There no reason to use a function for this.
```suggestion
	const miniaudioHeaderPath = "zig-out/include/miniaudio.h";
```